src/layouts/partials/icons/calendar-event.svg

```diff
@@ -1,4 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg"{{ with .width }} width="{{ . }}"{{ end }}{{ with .height }} height="{{ . }}"{{ end }}{{ with .class }} class="{{ . }}"{{ end }} viewBox="0 0 16 16" role="img" fill="currentColor">
```

Comment:
it's fine not to have the `role="img"` on these, as without a matching `<title>` or `aria-label="..."` in the `<svg>` it's pointless anyway (and we do provide accessible names separately when these are used)